_docs-2/troubleshooting/tools.md

```diff
@@ -152,3 +152,116 @@ An XML dump file can be later used to restore ZooKeeper.
     $ accumulo org.apache.accumulo.server.util.RestoreZookeeper --overwrite < dump.xml
 
 This command overwrites ZooKeeper so take care when using it. This is also why it cannot be called using `accumulo-util`.
+
+# zoo-info-viewer (new in 2.1)
+
+View Accumulo information stored in ZooKeeper in a human-readable format.  The utility can be run without an Accumulo 
+instance. If an instance id or name is not provided on the command line, the instance will be read from
+HDFS, otherwise only a running ZooKeeper instance is required to run the command.
+
+To run the command:
+
+    $ accumulo zoo-info-viewer [--instanceId id | --instanceName name] [mode-options] [--outfile filename]
+
+    mode options:
+    --print-instances
+    --print-id-map
+    --print-properties [--system] [-ns | --namespaces list] [-t | --tables list]
+
+## mode: print instances
+The instance name(s) and instance id(s) are stored in ZooKeeper. To see the available name, id mapping run:
+
+    $ accumulo zoo-info-viewer  --print-instances
+
+    -----------------------------------------------
+    Report Time: 2022-05-31T21:07:19.673258Z
+    -----------------------------------------------
+    Instances (Instance Name, Instance ID)
+    test_a=1111465d-b7bb-42c2-919b-111111111111
+    test_b=2222465d-b7bb-42c2-919b-222222222222
+    uno=9cc9465d-b7bb-42c2-919b-ddf74b610c82
+    
+    -----------------------------------------------
+
+## mode: print id-map
+If a shell is not available or convent, the zoo-info-viewer can provide the same 
```

Comment:
I'm not sure what you mean by `convent` here.

Reply:
spell check correction for my version of `convenient`